src/msw/graphicsd2d.cpp

```diff
@@ -2704,6 +2710,17 @@ wxCOMPtr<IDWriteTextLayout> wxD2DFontData::CreateTextLayout(const wxString& text
         textLayout->SetStrikethrough(true, textRange);
     }
 
+    if (m_spacing)
+    {
+#if WINVER >= 0x602

```

Comment:
BTW, we really shouldn't use `WINVER` checks in our code, see the comment in `wx/msw/winver.h`.

Reply:
Ok so just the runtime check is enough? I'll do the change.